libs/vm-modules/src/math/bignumber.cpp

```diff
@@ -288,12 +256,7 @@ void UInt256Wrapper::Subtract(Ptr<Object> &lhso, Ptr<Object> &rhso)
     lhs->number_ -= rhs->number_;
     return;
   }
-  if (rhs->IsTemporary())
-  {
-    rhs->number_ -= lhs->number_;
```

Comment:
Removed this block, because it substration is not commutative operation => rhs & lhs can NOT be swapped